### ISSUE TITLE
(Yt Music)feat: Patch to hide samples button

### ISSUE THREAD

**Issue by DopeDepot** [ReVanced/revanced-patches#2900]
### Application

Youtube Music 

### Issue

Youtube music has introduced the Samples just like shorts on YouTube. 

### Patch

There should be a patch to hide the samples button from bottom panel somewhat like the hide shorts button patch for youtube

### Motivation

Yt music will stay a music platform rather than reels or tiktok clone 

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by PraveenAkio** [comment]
Yeah it's annoying for me too.. please release the patch to fix this.
